### PR TITLE
Dialogue loading fix

### PR DESCRIPTION
**Pull request by NancokPS2**
Fixes exports attempting to load the ".import" files of the dialogue resources.

### CODE REVIEW

**Comment by jonathaneeckhout** [comment]
Step I did to test this:

Export project to different folder.

run server and gateway via normal editor

run the client as the exported executable in the other directory.

When trying to interact with the milkady I still get:

```
SCRIPT ERROR: Invalid call. Nonexistent function 'duplicate' in base 'Nil'.
          at: DialogueSynhcronizerComponent.sync_invoke_response (res://scripts/components/networking/dialoguesynhcronizercomponent/DialogueSynchronizerComponent.gd:55)
```

File: scripts/singletons/jdungeon.gd [hunk 1]
```diff
@@ -177,12 +177,18 @@ func register_class_resource(charclass_class: String, class_res_path: String):
 
 ## There'll be a LOT of dialogue, so loading each manually could not only dramatically increase load times, but it could also make errors more likely.
 func register_dialogue_all(dialogue_folder: String):
-	for file: String in DirAccess.get_files_at(dialogue_folder):
+	var allFiles: PackedStringArray = DirAccess.get_files_at(dialogue_folder)
```
Comment (jonathaneeckhout) [inline scripts/singletons/jdungeon.gd:180]: I am pretty sure the problem is with this function.

This function will look at the files located at "res://scenes/ui/dialogue/Dialogues/"

However this directory location does not exist in a exported build.

So there are 2 options here:

1) make sure this directory is present in the exported builds
2) go for KISS and load the res

Comment (NancokPS2) [reply scripts/singletons/jdungeon.gd:180]: You're sure that not even the .import files exist?  
If not, then i'll just have to embrace the KISS

**Comment by jonathaneeckhout** [comment]
Tested and works like charm. Merging.

Thanks for your contribution!

**Pull request merged**